bigtop-tests/smoke-tests/hive/HiveJdbcGeneralTest.java

```diff
@@ -0,0 +1,87 @@
+

```

Comment:
The first two lines are blank - no need to have them.
Instead, please put the ALv2 header, like other java files have